### PR TITLE
Aizad/77017/renaming synthetic to derived

### PR DESCRIPTION
**Pull request by aizad-deriv**
Task: Renaming Synthetic to Derived in MT5 dashboard

### CODE REVIEW

**Review by akmal-binary — APPROVED** [review]

File: src/templates/app/user/metatrader.jsx [hunk 1]
```diff
@@ -254,17 +266,6 @@ const Metatrader = () => (
                         ]}
                     /> */}
```
Comment (suisin-deriv) [inline src/templates/app/user/metatrader.jsx:267]: Remove commented code

Comment (aizad-deriv) [reply src/templates/app/user/metatrader.jsx]: Resolved issues 😀 👍🏼

**Review by akmal-binary — APPROVED** [review]

**Review by suisin-deriv — DISMISSED** [review]
LGTM

**Review by suisin-deriv — DISMISSED** [review]
LGTM

**Review by thisyahlen-deriv — APPROVED** [review]

**Review by suisin-deriv — APPROVED** [review]
LGTM

**Review by matin-binary — APPROVED** [review]

**Pull request merged**